src/main/webapp/scripts/deletescript.js

```diff
@@ -12,11 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// TODO: Start fetch methods to communicate with backend
-
-// Initialize the modal 
+// Initialize the modal.
 $('#deleteForm').on('submit', function(e){
   $('#deleteModal').modal('show');
   e.preventDefault();
 });
 
+
+// Call the doDelete method in the VendorServlet to delete the given vendorID.
+async function deleteVendor(form) {
+  const vendorID = form.vendorID.value;
+  const response = await fetch('/VendorServlet?vendorID=' + vendorID, 
+    { method: "DELETE"});
+  const deleteMessage = await response.json();
+
+  // Write a message if the delete was sucessful or not.
```

Comment:
Maybe comment "show an alert indicating whether or not the deletion succeeded"

Reply:
Sure